### ISSUE TITLE
Enhancement: Improved sidebar drowdown

### ISSUE THREAD

**Issue by njannink** [stsrki/Blazorise#1052]
Hello something that we would like for the dropdown menu's in the sidebar is to also add buttons. Like for example in Slack:

![image](https://user-images.githubusercontent.com/3843546/86132706-e194fb80-baac-11ea-80be-6bbdea12acab.png)

Enhancements:
- Possibility to add buttons
- Change location of dropdown icon to front or back (float: left or right)
- Change the dropdown icon into an actual button. now it's not clickable and can't be styled like a button
- Add the option to make the label a hyperlink (like the split dropdown)

**Comment by stsrki** [comment]
I don't have Slack installed. Can you record a video to show the behavior of the requested menu?

**Comment by stsrki** [comment]
PS. @MitchellNZ You're en expert in this area. Maybe give some comment?

**Comment by MitchellNZ** [comment]
These features would definitely make the vertical bar more powerful. But some of these might be tricky to get right across breakpoints and bar modes!

I already have quite a few things on the list for 0.9.2 bar improvements..
Perhaps we could revisit this after?

Otherwise, your welcome to work on this and submit a PR alongside my bar changes for 0.9.2!

**Comment by stsrki** [comment]
I agree this is not priority for now. Let us put it on the list for later. In the meantime anyone can try to implement it and submit a PR.
